spark3/src/main/java/org/apache/iceberg/spark/SparkSessionCatalog.java

```diff
@@ -240,8 +244,30 @@ public void renameTable(Identifier from, Identifier to) throws NoSuchTableExcept
     }
   }
 
+  public boolean isHiveCatalogConfigValid(CaseInsensitiveStringMap options, StringBuilder errorMsg) {
+    String hadoopConfUri = SparkContext.getOrCreate().conf().get("spark.hadoop.hive.metastore.uris");
```

Comment:
This can't call `SparkContext.getOrCreate()`. There is a `SparkContext.active()` that you can use instead.

Reply:
changed